web/news.md

```diff
@@ -4,6 +4,13 @@ summary: Important events happening.
 ---
 # News
 
+### 2022-10-29 - Changing the default environment from MINGW64 to UCRT64
+
+About 1.5 years ago we started adding a new variant of the MINGW64 environment called [UCRT64](./docs/environments), which uses the Universal CRT instead of
```

Comment:
I think you need another ., right now it gives 404 :p

Reply:
thanks, fixed!